changelogs/unreleased/bump-metrics-to-3370f85.md

```diff
@@ -0,0 +1,3 @@
+## feature/lua
+
+* Added a `tnt_election_leader_idle` metric to built-in `metrics`.
```

Comment:
Technically speaking, we can't add features into a bugfix release.

However, I guess that your motivation to propose this update to `release/3.0` is about testing changes: something may broke if we don't push it here. Am I right? I would like if you reveal more details about a motivation *of the backport*.

Taking it into account, we can prepare a special `tarantool-3.0` branch in the metrics repository, push needed testing fixes to it and update the submodule in `release/3.0` on the head of this new branch.

OTOH, such a branching requires extra effort and attention, it costs. So, I would propose an alternative.

First, why we don't add features to bugfix releases? The usual answer is stability, but it is not the whole story. There are quite isolated features that unlikely degrade any existing functionality, but we still don't want to have them in a bugfix release.

The reason is that we want to make the branching model clear for a user. A bugfix release solves problems in existing functionality. A feature release brings new functionality. If we mix these kinds of changes in the release notes, the branching model looks like a mess.

The solution is to be fair in motivation of an update and broadcast it via changelog/release notes.

Let's compare.

```markdown
## Release 3.0.2

### Functionality added

* `metrics` submodule is updated to X to bring the new metric Y.
```

```markdown
## Release 3.0.2

### Testing

* `metrics` submodule is updated to X for compatibility with recent version of
  Y, which is used in our testing/CI.
```

The top level section `Testing` [was added](https://github.com/tarantool/tarantool/pull/9451) for this kind of changes.

You can [check out](https://github.com/tarantool/tarantool-ee/pull/725) my example of such a change.

Reply:
See updated changelog